UPGRADE-3.4.md

```diff
@@ -72,6 +72,15 @@ DependencyInjection
  * The `ResolveDefinitionTemplatesPass` class is deprecated and will be removed in 4.0.
    Use the `ResolveChildDefinitionsPass` class instead.
 
+ * Parameter `container.dumper.inline_class_loader` was added in order to provide
+   control in the way used to dump the loaded classes. If configured with value `true`,
+   they will be dumped inline:
```

Comment:
Unless you're using a custom autoloader, you should enable the `container.dumper.inline_class_loader` parameter. This can dramatically improve DX by reducing the time to load classes in the "dev" environment.

Reply:
Could we use "drastically" instead of "dramatically"?

Reply:
I think `DependencyInjection` component doesn't know about environments. Could we replace "classes in the "dev" environment" with something like "classes when the `DebugClassLoader` is enabled"? However, I don't know if this is the only change which has impact on the performance improvement.

Reply:
sure :)

Reply:
Talking about DebugClassLoader might be a bit abstract for ppl, that's why I mentioned the "dev" env.

> this is the only change which has impact on the performance improvement

bypassing the "prod" autoloader is also beneficial, but by a quite thin margin

Reply:
I agree, but IMO referencing the "environment" concept here goes against using `DependencyInjection` component decoupled from the framework.

Reply:
this is about the parameter, which means configuring the component, thus by definition: *for integration with something else*
I wouldn't mind personally...

Reply:
Precisely, _something else_ should be anything consuming this dependency.

Reply:
I've reworded the description in order to address your suggestion: https://github.com/symfony/symfony/pull/26276/files#diff-8c6ccda31d4a8303738b07ac0f325881R75